content/network/drivers/overlay.md

```diff
@@ -123,8 +121,7 @@ remove the `ingress` network.
 
 During the time that no `ingress` network exists, existing services which do not
 publish ports continue to function but are not load-balanced. This affects
-services which publish ports, such as a WordPress service which publishes port
```

Comment:
Deleted the last part of this because the same example is provided in the following line but can restore if needed.